### PR TITLE
Document Address class

### PR DESCRIPTION
**Pull request by firstred**
| Questions | Answers |
| --- | --- |
| Branch? | `develop` |
| Description? | Upgrading the PHPDoc comments of this class, using the latest PS standard. Also getting the Franglais out of the comments, getting rid of things my Symfony code sniffer nags about, etc. It might have some additional `todo` tags, depending on whether the code was comprehensible or not. |
| Type? | improvement |
| Category? | CO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | N/A |
| How to test? | N/A |

### CODE REVIEW

File: classes/Address.php [hunk 1]
```diff
@@ -108,33 +113,34 @@ class AddressCore extends ObjectModel
...
+            'phone' => array('type' => self::TYPE_STRING, 'validate' => 'isPhoneNumber', 'size' => 32),
+            'phone_mobile' => array('type' => self::TYPE_STRING, 'validate' => 'isPhoneNumber', 'size' => 32),
+            'dni' => array('type' => self::TYPE_STRING, 'validate' => 'isDniLite', 'size' => 16),
+            'deleted' => array('type' => self::TYPE_BOOL, 'validate' => 'isBool', 'copy_post' => false),
+            'date_add' => array('type' => self::TYPE_DATE, 'validate' => 'isDate', 'copy_post' => false),
+            'date_upd' => array('type' => self::TYPE_DATE, 'validate' => 'isDate', 'copy_post' => false),
         ),
     );
 
     protected $_includeContainer = false;
```
Comment (mickaelandrieu) [inline classes/Address.php]: did you ignore it intentionaly ? If yes, why ?

Comment (firstred) [inline classes/Address.php]: No idea what it means

Comment (firstred) [inline classes/Address.php]: Can't find usages as well :confused:

Comment (mickaelandrieu) [inline classes/Address.php]: Ok, let's merge.

If we can remove it, we will do it in a pull request with this specific purpose 👍

**Comment by mickaelandrieu** [comment]
Thanks as always!

**Pull request merged**